### DISCUSSION TITLE
[Enhancement]: Add Lyrics Display Option to Miniplayer

### DISCUSSION THREAD

**Discussion by SMMdesign** [ciderapp/Cider#618] [Feature Request]
### The feature you would like

I would like to see an option to show the scrolling lyrics in the Miniplayer instead of the album art. 

Current:
![image](https://user-images.githubusercontent.com/100661938/159383984-22584cfe-ddfa-45d8-bd3f-9659fba17e18.png)

After:
![image](https://user-images.githubusercontent.com/100661938/159383787-76ae5ff6-e16b-482d-89b2-a818568a4460.png)

### Alternatives and Other Information

Possibly have an option to change the font size too

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by tva2000hn** [comment]
Use the lyrics plugin for this

**Comment by SMMdesign** [comment]
I apologize if I'm looking in the wrong place, but the wiki page for plugins seems to be essentially blank for me. Can you point me in the right direction to look for the plugin you're talking about?

https://github.com/ciderapp/Cider/wiki/Plugins

**Comment by yazninja** [comment]
> I apologize if I'm looking in the wrong place, but the wiki page for plugins seems to be essentially blank for me. Can you point me in the right direction to look for the plugin you're talking about?

Cider Button > Settings > Experimental > Explore Github Plugins
then after installing the plugin you will see an option in the Cider Button called Plug-in
Enable it from there
![image](https://user-images.githubusercontent.com/71800112/159451965-1b8b2927-4552-44f4-af53-97201bc59549.png)

**Comment by SMMdesign** [comment]
I don't seem to have an option for that, only "Unknown Sources" which I don't notice changes anything. Also my Cider button is on the left? I'm using the version downloaded from the Windows store, could that be the reason? (Changing to the Development option also doesn't seem to change anything I can see.)

![image](https://user-images.githubusercontent.com/100661938/159597799-e7697ced-ecc1-43e3-a915-7aa4ac38ba7b.png)

**Comment by yazninja** [comment]
>  I'm using the version downloaded from the Windows store, could that be the reason? 

I'm sorry, but the plugin feature seems to not be out yet on the Windows store version.

> (Changing to the Development option also doesn't seem to change anything I can see.)

This button does nothing on the Windows Store version as you will be using the MS Store to update your apps.

**Reply by CrterBrian** [reply]
So which download method should I use?
